goinp/goinp.go

```diff
@@ -116,35 +144,47 @@ func askOptionsWithReader(title string, defaultValue string, optional bool, read
 	}
 
 	if title != "" {
```

Comment:
i would not do auto selection by default, i would let the client decide about it, this is a core package for fetching user inputs

Reply:
if title != "" { this is just a check to not to print the title in line 147 if there was no any